### ISSUE TITLE
Fails on nested dictionaries/sets

### ISSUE THREAD

**Issue by Nico314159** [xXCrash2BomberXx/Bython#4]
All of the following create syntax errors when compiled:
```cs
my_dict = {
    "a": {1, 2, 3}
}
```

```cs
my_dict =  {
    "b": {"c": 5}
}
```

```cs
x = {{1, 2, 3}, 4, 5}
```

**Comment by Nico314159** [comment]
@xXCrash2BomberXx please fix soon, it really breaks my workflow

**Comment by xXCrash2BomberXx** [comment]
So the immediate issue I see with the first one is the lack of a comma. Since the language now uses braces as context specifiers, the comma declares it as a dictionary/set instead of a context meaning it must be written as follows
```
my_dict = {
    "a": {1, 2, 3},
}
```

The same with the second
```
my_dict =  {
    "b": {"c": 5,},
}
```
And the third is both a bug with the build I had and a syntax error with Python meaning it has to be written as below since sets can only contain serializable data
```
x = [{1, 2, 3}, 4, 5]
```

Overall, appreciate the bug report and the most recent version should fix the issue. If you find anything else, let me know!

**Comment by Nico314159** [comment]
Thanks

> So the immediate issue I see with the first one is the lack of a comma. Since the language now uses braces as context specifiers, the comma declares it as a dictionary/set instead of a context meaning it must be written as follows
> 
> ```
> my_dict = {
>     "a": {1, 2, 3},
> }
> ```
> 
> The same with the second
> 
> ```
> my_dict =  {
>     "b": {"c": 5,},
> }
> ```
> 
> And the third is both a bug with the build I had and a syntax error with Python meaning it has to be written as below since sets can only contain serializable data
> 
> ```
> x = [{1, 2, 3}, 4, 5]
> ```
> 
> Overall, appreciate the bug report and the most recent version should fix the issue. If you find anything else, let me know!

Thanks, stuff works great now. Do you also think you could fix/change #3 too?
